components/IntakeForm/index.js

```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
+import { TrackerForm } from "./IntakeForm.styled";
+import { SubmitButton } from "../MachineForm/MachineForm.Styled";
+import { FormInput } from "../MachineForm/MachineForm.Styled";
+
+export default function IntakeForm({ onsubmit }) {
+  const [carbs, setCarbs] = useState("");
+  const [protein, setProtein] = useState("");
+  const [fat, setFat] = useState("");
+  const [kcal, setKcal] = useState("");
+
+  const handleCarbsChange = (event) => {
+    const value = event.target.value;
+    setCarbs(value);
+    calculateKcal(value, protein, fat);
+  };
+
+  const handleProteinChange = (event) => {
+    const value = event.target.value;
+    setProtein(value);
+    calculateKcal(carbs, value, fat);
+  };
+
+  const handleFatChange = (event) => {
+    const value = event.target.value;
+    setFat(value);
+    calculateKcal(carbs, protein, value);
+  };
+  const calculateKcal = (carbs, protein, fat) => {
+    const calculatedKcal = carbs * 4 + protein * 4 + fat * 9;
+    setKcal(calculatedKcal);
+  };
+
+  return (
+    <>
+      <TrackerForm onSubmit={onsubmit}>
+        <h2>My Intake</h2>
+        <label htmlFor="kcal">Kcal: </label>
+        <FormInput type="number" name="kcal" id="kcal" value={kcal} readOnly />
+        <label htmlFor="InCarbs">Carbs: </label>
+        <FormInput
+          type="number"
+          name="carbs"
+          id="carbs"
+          value={carbs}
+          onChange={handleCarbsChange}
+          placeholder="in grams"
+          required
+        />
+        <label htmlFor="InProtein">Protein: </label>
+        <FormInput
+          type="number"
+          name="protein"
+          id="protein"
+          value={protein}
+          onChange={handleProteinChange}
+          placeholder="in grams"
+          required
+        />
+        <label htmlFor="InFat">Fat: </label>
```

Comment:
The `htmlFor` refers to the `id` on the input element. 

So in this case i think it should be `htmlFor="fat"`